app/styles/ui/_merge-status.scss

```diff
@@ -17,7 +17,6 @@
 
     &:after {
```

Comment:
I don't think we need this whole `:after` element now that we aren't using the rule anywhere